### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
NFC, so none of the below seem to apply.

- [ ] Tested on all platforms changed
- [ ] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [ ] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [ ] Created or updated an example program if it would help users understand this functionality
- [ ] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

### CODE REVIEW

**Comment by kchibisov** [comment]
Could you merge all your typos PRs into a single one?

**Comment by waywardmonkeys** [comment]
Do you want the separate commits in a single PR or merge to a single commit? The changes not included in this one are ones that change code rather than only comments…

**Comment by kchibisov** [comment]
Everything as a single commit. it's all internal changes, so doesn't really matter.

File: CHANGELOG.md [hunk 1]
```diff
@@ -386,7 +386,7 @@ Unreleased` header.
 - Added `Window::set_transparent` to provide a hint about transparency of the window on Wayland and macOS.
 - On macOS, fix the mouse buttons other than left/right/middle being reported as middle.
 - On Wayland, support fractional scaling via the wp-fractional-scale protocol.
-- On web, fix removal of mouse event listeners from the global object upon window distruction.
+- On web, fix removal of mouse event listeners from the global object upon window destruction.
```
Comment (daxpedda) [inline CHANGELOG.md:389]:
```suggestion
- On Web, fix removal of mouse event listeners from the global object upon window destruction.
```
While we are at it.

**Review by daxpedda — APPROVED** [review]
Thank you for doing this!

File: CHANGELOG.md (hunk 1 above)
Comment (waywardmonkeys) [reply CHANGELOG.md]: Applied that edit and one other instance of the same edit.

**Comment by waywardmonkeys** [comment]
Just noticed this had conflicts preventing a merge. Rebased and force pushed.

**Pull request merged**